examtool/examtool/api/download.py

```diff
@@ -1,189 +1,12 @@
-
-        def out(text):
-            pdf.multi_cell(
-                200,
-                5,
-                txt=text.encode("latin-1", "replace")
-                .decode("latin-1")
-                .replace("\t", " " * 4),
-                align="L",
-            )
-
-    student_question_lookup = {q["id"]: q for q in student_questions}
-
-    for question in template_questions:
-        pdf.add_page()
-        out("\nQUESTION")
-        if substitute_in_question_text:
-            question_for_text = student_question_lookup.get(question["id"], question)
-        else:
-            question_for_text = question
-        for line in question_for_text["text"].split("\n"):
-            out(line)
-
-        out("\nANSWER")
-
-        if question.get("type") in ["multiple_choice", "select_all"]:
-            selected_options = response.get(question["id"], [])
-            if question.get("type") == "multiple_choice" and not isinstance(
-                selected_options, list
-            ):
-                selected_options = [selected_options]
-            available_options = sorted(
-                [(i, option["text"]) for i, option in enumerate(question["options"])]
-            )
-            if question["id"] not in student_question_lookup:
-                student_options = sorted(
-                    [
-                        (option.get("index", i), option["text"])
-                        for i, option in enumerate(question["options"])
-                    ]
-                )
-            else:
-                student_options = sorted(
-                    [
-                        (option.get("index", i), option["text"])
-                        for i, option in enumerate(
-                            student_question_lookup[question["id"]]["options"]
-                        )
-                    ]
-                )
-
-            available_options = [
-                (*available_option, option)
-                for available_option, (j, option) in zip(
-                    available_options, student_options
-                )
-            ]
-
-            available_options.sort(key=lambda x: x[1])
-
-            for i, template, option in available_options:
-                if option in selected_options:
-                    template = template
-                    out("[X] " + template)
-                else:
-                    out("[ ] " + template)
-        else:
-            for line in response.get(question["id"], "").split("\n"):
-                out(line)
-
-        out("\nAUTOGRADER")
-        if question["id"] in student_question_lookup and question["id"] in response:
-            out(
-                grade(
-                    email, student_question_lookup[question["id"]], response, dispatch
-                )
-            )
-        elif question["id"] not in student_question_lookup:
-            out("STUDENT DID NOT RECEIVE QUESTION")
-        else:
-            out("")
-
-    return pdf
-
-
-def export(
```

Comment:
Removing this function will require a change in line 532 of the api gradescope_autograde class as it relies on this export. Not a hard change but a change nonetheless.